packages/api/src/index.ts

```diff
@@ -103,6 +113,7 @@ void (async () => {
   .then(() => {
     app.listen(port, () => {
       logger.info(`🚀 Listening at http://localhost:${port}`);
+      contributionPollTimer(3600000);
```

Comment:
The problem with using static times like this is that the implementation above doesn't make a call immediately and if you deploy once an hour, you'll _**never**_ make a new query to update those things. Same thing goes if the app restarts once per hour...

What we should be doing here (and sorry for not calling this out earlier) is using something like [`cron`](https://www.npmjs.com/package/cron) to manage things and keep the polling to specified times. This allows us to schedule them during downtime and will also help mitigate the impact of redeploying or restarting. When you start figuring out how often to poll, use [`Crontab`](https://crontab.guru) to help with the schedule expressions. [Here's an example of once an hour on the hour](https://crontab.guru/#0_*_*_*_*).

As a general comment, I always prefer making constants to make numbers like this more readable. If we were to stick with this approach, I'd say you should add something like this to the top of the file:
```typescript
const contributionPollInterval = 1 * 60 * 60 * 1000; // 1 hour
```

Reply:
done